src/components/TeacherViews/LessonOverview/LessonOverview.view.tsx

```diff
@@ -274,6 +283,50 @@ export function LessonOverview({ lesson }: LessonOverviewProps) {
             </GridArea>
             <OakGridArea $colSpan={[12, 9]}>
               <OakFlex $flexDirection={"column"} $position={"relative"}>
+                {pageLinks.find((p) => p.label === "Lesson guide") &&
+                  lessonGuideUrl && (
+                    <LessonItemContainer
+                      isSpecialist={isSpecialist}
+                      ref={lessonGuideSectionRef}
+                      title={"Lesson guide"}
+                      //Defaulted to false until download ticket implementation
+                      downloadable={false}
+                      // Avo types need to be updated to include lesson guide
+                      // onDownloadButtonClick={() => {
+                      //   trackDownloadResourceButtonClicked({
+                      //     downloadResourceButtonName: "lesson guide",
+                      //   });
+                      // }}
+                      slugs={slugs}
+                      anchorId="lesson-guide"
+                      pageLinks={pageLinks}
+                    >
+                      <OakBox
+                        $width={"100%"}
+                        $ba={"border-solid-m"}
+                        style={{ height: "100%" }}
+                      >
+                        <AspectRatio ratio={"16:9"}>
+                          <iframe
+                            data-testid="lesson-guide-iframe"
+                            src={`${previewLessonGuideUrl}`}
+                            title={`lesson guide: ${lessonTitle}`}
+                            width="auto"
+                            height="100%"
+                            style={{
+                              border: "none",
+                            }}
+                            //small render bug fix to make sure the iframe assumes 100% width. Docs are still in unpublished currently so temporary
```

Comment:
do we have these todos captured in a ticket?

Reply:
Will create a ticket for this now